airsonic-main/src/main/java/org/airsonic/player/controller/RecoverController.java

```diff
@@ -39,6 +37,11 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(RecoverController.class);
 
+    /** HTML snippet for inserting a reCAPTCHA. */
+    private static final String RECAPTCHA_TEMPLATE =
+        "<script src=\"https://www.google.com/recaptcha/api.js\" async defer></script>"
+        + "<div class=\"g-recaptcha\" data-sitekey=\"%s\"></div>";
+
```

Comment:
If this is static, it should probably just go in the jsp directly probably wrapped around an if conditional.